src/mainwindow.cpp

```diff
@@ -506,6 +506,8 @@ void MainWindow::initTreeWidgets() {
             &MainWindow::onMultipleSubfoldersSelected);
     connect(ui->noteSubFolderTreeWidget, &NoteSubFolderTree::currentSubFolderChanged, this,
             &MainWindow::onCurrentSubFolderChanged);
+    connect(ui->noteTreeWidget, &QTreeWidget::itemExpanded, ui->noteSubFolderTreeWidget, &NoteSubFolderTree::onItemExpanded);
```

Comment:
This only should be done in the experimental note tree mode!

Reply:
I'm not sure the notelist can be expanded if not in experimental note tree mode?

Reply:
No, the note list is a list by default.

Reply:
> This only should be done in the experimental note tree mode!

Do you want to add the check? There is a method for it.

Reply:
Done !

Reply:
Thank you!